### PR TITLE
Add Tuya switch TS0601 `_TZE204_ptaqh9tk` variant

### LINKED ISSUE #3099: [BUG]  Tuya TS0601 _TZE204_ptaqh9tk
### Bug description

Not work
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks.registry] Checking quirks for _TZE204_ptaqh9tk TS0601 (70:ac:08:ff:fe:c7:f7:59)
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.tuya.ts0601_switch.TuyaSingleSwitch_GP'>
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks] Fail because endpoint list mismatch: {1, 242} {1}

if delete '("_TZE204_ptaqh9tk", "TS0601")' from class TuyaSingleSwitch_GP and add to class TuyaSingleSwitchTO works

### Steps to reproduce

When adding a device 

### Expected behavior

so that it works

### Screenshots/Video

<details><summary>Screenshots/Video</summary>

image

</details>

### Device signature

<details><summary>Device signature</summary>

```json
{
  "node_descriptor": "NodeDescriptor(logical_type=<LogicalType.Router: 1>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.FullFunctionDevice|MainsPowered|RxOnWhenIdle|AllocateAddress: 142>, manufacturer_code=4098, maximum_buffer_size=82, maximum_incoming_transfer_size=82, server_mask=11264, maximum_outgoing_transfer_size=82, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=False, *is_full_function_device=True, *is_mains_powered=True, *is_receiver_on_when_idle=True, *is_router=True, *is_security_capable=False)",
  "endpoints": {
    "1": {
      "profile_id": "0x0104",
      "device_type": "0x0051",
      "input_clusters": [
        "0x0000",
        "0x0004",
        "0x0005",
        "0xef00"
      ],
      "output_clusters": [
        "0x000a",
        "0x0019"
      ]
    }
  },
  "manufacturer": "_TZE204_ptaqh9tk",
  "model": "TS0601",
  "class": "zigpy.device.Device"
}
```

</details>

### Diagnostic information

<details><summary>Diagnostic information</summary>

```json
{
  "home_assistant": {
    "installation_type": "Home Assistant Core",
    "version": "2024.3.3",
    "dev": false,
    "hassio": false,
    "virtualenv": true,
    "python_version": "3.12.2",
    "docker": false,
    "arch": "x86_64",
    "timezone": "Etc/UTF",
    "os_name": "Linux",
    "os_version": "5.14.0-362.24.1.el9_3.x86_64",
    "run_as_root": false
  },
  "custom_components": {},
  "integration_manifest": {
    "domain": "zha",
    "name": "Zigbee Home Automation",
    "after_dependencies": [
      "onboarding",
      "usb"
    ],
    "codeowners": [
      "@dmulcahey",
      "@adminiuga",
      "@puddly",
      "@TheJulianJES"
    ],
    "config_flow": true,
    "dependencies": [
      "file_upload"
    ],
    "documentation": "https://www.home-assistant.io/integrations/zha",
    "import_executor": true,
    "iot_class": "local_polling",
    "loggers": [
      "aiosqlite",
      "bellows",
      "crccheck",
      "pure_pcapy3",
      "zhaquirks",
      "zigpy",
      "zigpy_deconz",
      "zigpy_xbee",
      "zigpy_zigate",
      "zigpy_znp",
      "universal_silabs_flasher"
    ],
    "requirements": [
      "bellows==0.38.1",
      "pyserial==3.5",
      "pyserial-asyncio==0.6",
      "zha-quirks==0.0.112",
      "zigpy-deconz==0.23.1",
      "zigpy==0.63.4",
      "zigpy-xbee==0.20.1",
      "zigpy-zigate==0.12.0",
      "zigpy-znp==0.12.1",
      "universal-silabs-flasher==0.0.18",
      "pyserial-asyncio-fast==0.11"
    ],
    "usb": [
      {
        "vid": "10C4",
        "pid": "EA60",
        "description": "*2652*",
        "known_devices": [
          "slae.sh cc2652rb stick"
        ]
      },
      {
        "vid": "10C4",
        "pid": "EA60",
        "description": "*slzb-07*",
        "known_devices": [
          "smlight slzb-07"
        ]
      },
      {
        "vid": "1A86",
        "pid": "55D4",
        "description": "*sonoff*plus*",
        "known_devices": [
          "sonoff zigbee dongle plus v2"
        ]
      },
      {
        "vid": "10C4",
        "pid": "EA60",
        "description": "*sonoff*plus*",
        "known_devices": [
          "sonoff zigbee dongle plus"
        ]
      },
      {
        "vid": "10C4",
        "pid": "EA60",
        "description": "*tubeszb*",
        "known_devices": [
          "TubesZB Coordinator"
        ]
      },
      {
        "vid": "1A86",
        "pid": "7523",
        "description": "*tubeszb*",
        "known_devices": [
          "TubesZB Coordinator"
        ]
      },
      {
        "vid": "1A86",
        "pid": "7523",
        "description": "*zigstar*",
        "known_devices": [
          "ZigStar Coordinators"
        ]
      },
      {
        "vid": "1CF1",
        "pid": "0030",
        "description": "*conbee*",
        "known_devices": [
          "Conbee II"
        ]
      },
      {
        "vid": "0403",
        "pid": "6015",
        "description": "*conbee*",
        "known_devices": [
          "Conbee III"
        ]
      },
      {
        "vid": "10C4",
        "pid": "8A2A",
        "description": "*zigbee*",
        "known_devices": [
          "Nortek HUSBZB-1"
        ]
      },
      {
        "vid": "0403",
        "pid": "6015",
        "description": "*zigate*",
        "known_devices": [
          "ZiGate+"
        ]
      },
      {
        "vid": "10C4",
        "pid": "EA60",
        "description": "*zigate*",
        "known_devices": [
          "ZiGate"
        ]
      },
      {
        "vid": "10C4",
        "pid": "8B34",
        "description": "*bv 2010/10*",
        "known_devices": [
          "Bitron Video AV2010/10"
        ]
      }
    ],
    "zeroconf": [
      {
        "type": "_esphomelib._tcp.local.",
        "name": "tube*"
      },
      {
        "type": "_zigate-zigbee-gateway._tcp.local.",
        "name": "*zigate*"
      },
      {
        "type": "_zigstar_gw._tcp.local.",
        "name": "*zigstar*"
      },
      {
        "type": "_uzg-01._tcp.local.",
        "name": "uzg-01*"
      },
      {
        "type": "_slzb-06._tcp.local.",
        "name": "slzb-06*"
      }
    ],
    "is_built_in": true
  },
  "data": {
    "ieee": "**REDACTED**",
    "nwk": 43674,
    "manufacturer": "_TZE204_ptaqh9tk",
    "model": "TS0601",
    "name": "_TZE204_ptaqh9tk TS0601",
    "quirk_applied": false,
    "quirk_class": "zigpy.device.Device",
    "quirk_id": null,
    "manufacturer_code": 4098,
    "power_source": "Mains",
    "lqi": 171,
    "rssi": null,
    "last_seen": "2024-04-09T19:04:06",
    "available": true,
    "device_type": "Router",
    "signature": {
      "node_descriptor": "NodeDescriptor(logical_type=<LogicalType.Router: 1>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.FullFunctionDevice|MainsPowered|RxOnWhenIdle|AllocateAddress: 142>, manufacturer_code=4098, maximum_buffer_size=82, maximum_incoming_transfer_size=82, server_mask=11264, maximum_outgoing_transfer_size=82, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=False, *is_full_function_device=True, *is_mains_powered=True, *is_receiver_on_when_idle=True, *is_router=True, *is_security_capable=False)",
      "endpoints": {
        "1": {
          "profile_id": "0x0104",
          "device_type": "0x0051",
          "input_clusters": [
            "0x0000",
            "0x0004",
            "0x0005",
            "0xef00"
          ],
          "output_clusters": [
            "0x000a",
            "0x0019"
          ]
        }
      },
      "manufacturer": "_TZE204_ptaqh9tk",
      "model": "TS0601"
    },
    "active_coordinator": false,
    "entities": [
      {
        "entity_id": "update.tze204_ptaqh9tk_ts0601_firmware",
        "name": "_TZE204_ptaqh9tk TS0601"
      }
    ],
    "neighbors": [],
    "routes": [],
    "endpoint_names": [
      {
        "name": "SMART_PLUG"
      }
    ],
    "user_given_name": null,
    "device_reg_id": "77a067a1128837bb3ed0af881a74705e",
    "area_id": null,
    "cluster_details": {
      "1": {
        "device_type": {
          "name": "SMART_PLUG",
          "id": 81
        },
        "profile_id": 260,
        "in_clusters": {
          "0x0000": {
            "endpoint_attribute": "basic",
            "attributes": {
              "0x0001": {
                "attribute_name": "app_version",
                "value": 70
              },
              "0x0004": {
                "attribute_name": "manufacturer",
                "value": "_TZE204_ptaqh9tk"
              },
              "0x0005": {
                "attribute_name": "model",
                "value": "TS0601"
              }
            },
            "unsupported_attributes": {}
          },
          "0x0004": {
            "endpoint_attribute": "groups",
            "attributes": {},
            "unsupported_attributes": {}
          },
          "0x0005": {
            "endpoint_attribute": "scenes",
            "attributes": {},
            "unsupported_attributes": {}
          },
          "0xef00": {
            "endpoint_attribute": null,
            "attributes": {},
            "unsupported_attributes": {}
          }
        },
        "out_clusters": {
          "0x0019": {
            "endpoint_attribute": "ota",
            "attributes": {},
            "unsupported_attributes": {
              "0x0002": {
                "attribute_name": "current_file_version"
              }
            }
          },
          "0x000a": {
            "endpoint_attribute": "time",
            "attributes": {},
            "unsupported_attributes": {}
          }
        }
      }
    }
  }
}
```

</details>

### Logs

<details><summary>Logs</summary>

```python
2024-04-09 18:59:25.139 DEBUG (MainThread) [zigpy.application] Device is initialized <Device model='TS0601' manuf='_TZE204_ptaqh9tk' nwk=0xAA9A ieee=70:ac:08:ff:fe:c7:f7:59 is_initialized=True>
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks.registry] Checking quirks for _TZE204_ptaqh9tk TS0601 (70:ac:08:ff:fe:c7:f7:59)
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.tuya.ts0601_switch.TuyaSingleSwitch_GP'>
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks] Fail because endpoint list mismatch: {1, 242} {1}
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.xbee.xbee_io.XBeeSensor'>
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks] Fail because endpoint list mismatch: {232, 230} {1}
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.xbee.xbee3_io.XBee3Sensor'>
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks] Fail because endpoint list mismatch: {232, 230} {1}
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.tuya.ts0201.MoesTemperatureHumidtySensorWithScreen'>
2024-04-09 18:59:25.140 DEBUG (MainThread) [zigpy.quirks] Fail because device_type mismatch on at least one endpoint
2024-04-09 18:59:25.141 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.smartthings.tag_v4.SmartThingsTagV4'>
2024-04-09 18:59:25.141 DEBUG (MainThread) [zigpy.quirks] Fail because device_type mismatch on at least one endpoint
2024-04-09 18:59:25.141 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.smartthings.multi.SmartthingsMultiPurposeSensor'>
2024-04-09 18:59:25.141 DEBUG (MainThread) [zigpy.quirks] Fail because device_type mismatch on at least one endpoint
2024-04-09 18:59:25.141 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.netvox.z308e3ed.Z308E3ED'>
2024-04-09 18:59:25.141 DEBUG (MainThread) [zigpy.quirks] Fail because device_type mismatch on at least one endpoint
2024-04-09 18:59:25.141 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.gledopto.soposhgu10.SoposhGU10'>
2024-04-09 18:59:25.141 DEBUG (MainThread) [zigpy.quirks] Fail because endpoint list mismatch: {11, 13} {1}
```

</details>

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by VictorPavlushin**
## Proposed change
Tuya TS0601 _TZE204_ptaqh9tk  device does not work in this class
I bought it here, but now the page is not working: https://a.aliexpress.com/_Dk7dDPx

## Additional information
Fixes #3099
debug the log in the attached file:
[log.txt](https://github.com/zigpy/zha-device-handlers/files/14914132/log.txt)

## Checklist

- [ x ] The changes are tested and work correctly
- [ ] `pre-commit` checks pass / the code has been formatted using Black
- [ ] Tests have been added to verify that the new code works

### CODE REVIEW

File: zhaquirks/tuya/ts0601_switch.py [hunk 1]
```diff
@@ -127,7 +128,6 @@ class TuyaSingleSwitch_GP(TuyaSwitch):
         MODELS_INFO: [
             ("_TZE200_gbagoilo", "TS0601"),  # reported in #1634
             ("_TZE204_6fk3gewc", "TS0601"),
-            ("_TZE204_ptaqh9tk", "TS0601"),  # reported in #2780
```
Comment (javicalle) [inline zhaquirks/tuya/ts0601_switch.py]: Don't remove the model declaration here.
It can be defined in both classes.

**Review by javicalle — CHANGES_REQUESTED** [review]

**Comment by VictorPavlushin** [comment]
@javicalle Ready

File: zhaquirks/tuya/ts0601_switch.py (hunk 1 above)
Comment (VictorPavlushin) [reply zhaquirks/tuya/ts0601_switch.py]: Ready

**Review by javicalle — APPROVED** [review]
LGTM

File: zhaquirks/tuya/ts0601_switch.py [hunk 2]
```diff
@@ -87,6 +87,7 @@ class TuyaSingleSwitchTO(TuyaSwitch):
             ("_TZ3000_uim07oem", "TS0601"),  # ¿1 or 4 gangs?
             ("_TZE200_wfxuhoea", "TS0601"),
             ("_TZE200_tviaymwx", "TS0601"),
+            ("_TZE204_ptaqh9tk", "TS0601"),  # reported in #2780
```
Comment (TheJulianJES) [inline zhaquirks/tuya/ts0601_switch.py]:
```suggestion
            ("_TZE204_ptaqh9tk", "TS0601"),  # reported in #3099
```

**Review by TheJulianJES — APPROVED** [review]
Thanks!

**Pull request merged**